src/pages/ShareCodePage.tsx

```diff
@@ -82,6 +84,8 @@ function ShareCodePage({report, policy}: ShareCodePageProps) {
     }, [report, currentUserPersonalDetails, isReport]);
 
     const title = isReport ? ReportUtils.getReportName(report) : currentUserPersonalDetails.displayName ?? '';
+    const platform = getPlatform();
+    const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
```

Comment:
If this hasn't been discussed previously, why didn't we follow the no specific platform conditions rule?

Reply:
I don't have the answer to that question. It is preexisting code, I think that's @robertjchen that did this feature.

I just didn't want to introduce potential side effects that may make this feature broken, since this PR aims to put back code that was present before.

(You've subscribed to this PR since I've chatted with you yesterday about `react-native-view-shot`)

Reply:
Oh right lol, had already forgotten

Reply:
I know its annoying, but if we do ship the isMobile condition we should state the reason why. The more we have, the more people will use it as justification to introduce more.

Reply:
Ah, this was from a while back- IIRC we added this hack due to limitations with the QR library, in order to get the feature working in time for a conference. 🤔 

Since there's no time pressure anymore- maybe we can look into fix this by seeing if the limitation still exists, and if we still need this check?

Reply:
I've just tested it @Julesssss and the limitation still exists.

We're not able to download on web platform.

Reply:
I feel like so yes @robertjchen.

I've attached a video test:

https://github.com/user-attachments/assets/f35a1c3c-ab2d-496c-a9dd-19a03483a448

Reply:
Curious what the JS console shows 👀

Reply:
Here are the errors:

<img width="815" alt="image" src="https://github.com/user-attachments/assets/87a7169d-59b4-4daa-8c39-897ea203f667">
<img width="677" alt="image" src="https://github.com/user-attachments/assets/a2f9610b-4d27-469e-b351-5fbe9dd15f14">

And the warnings:
<img width="695" alt="image" src="https://github.com/user-attachments/assets/60e2cb52-0d75-4cd3-a804-044af923f8ce">

Reply:
cc @Julesssss @robertjchen I think we still need to keep this check for now.

We'll remove the condition once https://github.com/Expensify/App/issues/19834 is done.

Reply:
:sadcat_okay: Yep, that would be the proper issue to handle this.

Could we make a big comment there noting that this is a bad thing and that it'll be addressed in https://github.com/Expensify/App/issues/19834 🙏

Reply:
Fair enough, thanks for investigating

Reply:
I've added comments to make sure we don't follow the same logic elsewhere in our codebase.

Thank you both @Julesssss @robertjchen for checking!